plenoptic/synthesize/eigendistortion.py

```diff
@@ -282,7 +282,7 @@ def _synthesize_exact(self) -> Tuple[Tensor, Tensor]:
 
         J = self.compute_jacobian()
         F = J.T @ J
-        eig_vals, eig_vecs = torch.symeig(F, eigenvectors=True)
+        eig_vals, eig_vecs = torch.linalg.eigh(F, UPLO="U")
```

Comment:
isn't `torch.linalg` only available starting in `torch` 1.10?

Reply:
Also, this seems unrelated to the Jenkins / GPU tests issue, and so probably shouldn't be part of this PR?

Reply:
I believe torch.linalg.eigh is still there in torch 1.8 which we require so I don't think this is an issue

Reply:
yeah as nikhil said, it works as of 1.8 so there's no harm